README.md

```diff
@@ -0,0 +1,46 @@
+# go bindings for Elektra
+
+This repository contains the low level ("kdb" subpackage) and high level (root package, TODO) go bindings for the elektra library.
+
+# Prerequisites
+
+Set your `PKG_CONFIG_PATH` environment variable to where your elektra.pc files are located (if at a nonstandard location).
+
+E.g.: `PKG_CONFIG_PATH=/usr/local/lib/pkgconfig`.
+
+## Execute tests
+Prerequisite: you have to have KDB and go installed on your machine.
+
+Execute all tests:
+`go test ./...`
+
+Execute tests of a package, e.g. kdb:
+`go test ./kdb`
+
+## Use Elektra in your application
+Just *go get* it like you are used to with go.
+
+`go get github.com/ElektraInitiative/go-elektra`
+
+In the future we will add a vanity import.
+
+To use it import it in your .go file (error handling was omitted for brevity):
+
+```go
```

Comment:
This is just markdown syntax to signal a go codeblock

Reply:
I mean that you add a `package main` as first line so that the code can actually be executed.

Please do not resolve conversations if it is unclear what I meant :slightly_smiling_face: